recipes/parsmurf/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+
+cmake src
+make -j 4
+
+#cmake src -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_LIBDIR=lib -DBUILD_SHARED_LIBS=ON
```

Comment:
Also, usually you do need a call like this

Reply:
when changing to `cmake src -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_LIBDIR=lib -DBUILD_SHARED_LIBS=ON` I get an error (see https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=181634&view=logs&j=10bda42a-d420-5c28-a43f-bc4e66405e5c&t=bef775d0-a1a2-55a3-9721-17147029baf0)